templates/statefulset-zabbix-server.yaml

```diff
@@ -74,6 +74,28 @@ spec:
             - name: {{ $item.name }}
               value: {{ $item.value | quote }}
             {{- end }}
+          volumeMounts:
+            {{- range $item := .Values.zabbixproxy.volumeMounts }}
```

Comment:
should these be `.Values.zabbixserver.volumeMounts`?

In other words should it be made so different volumes can be mounted for the zabbixproxy and the zabbixserver?

Reply:
Absolutely, I messed up. Fixed in c9a7db7a2e3d527348b7fdb06b38d346cf528000